src/sync.rs

```diff
@@ -46,15 +47,15 @@ pub fn sync_remote_to_local(local_project_dir_name: &str, config: &Config, ignor
         .arg(format!(
             "{remote_machine_name}:{project_dir_on_remote_machine}/",
             remote_machine_name = config.remote_machine_name,
-            project_dir_on_remote_machine = project_dir_on_remote_machine(local_project_dir_name))
+            project_dir_on_remote_machine = project_dir_on_remote_machine(local_dir_absolute_path))
         )
         .arg("./");
 
     execute_rsync(&mut command)
 }
 
-fn project_dir_on_remote_machine(local_project_dir_name: &str) -> String {
-    format!("~/mainframer/{}", local_project_dir_name)
+pub fn project_dir_on_remote_machine(local_dir_absolute_path: &Path) -> String {
+    format!("~/mainframer/{}", local_dir_absolute_path.to_string_lossy())
```

Comment:
Could it be here `"~/mainframer//home/user/project"` as a result?

Reply:
Snap, good catch 👍 

Most of the time it's harmless to do so (you can check in your terminal), but I definitely did not intend that

Reply:
foxed, thanks again 😽